bindings/python/setup.py

```diff
@@ -448,7 +448,7 @@ def _configure_b2_with_distutils(self, python_binding_dir):
 
 
 setuptools.setup(
-    name="python-libtorrent",
```

Comment:
@AllSeeingEyeTolledEweSew I think this may be the only important part of this patch then